packages/xen-api/src/index.js

```diff
@@ -569,6 +572,14 @@ export class Xapi extends EventEmitter {
     throw new Error('no object with UUID: ' + uuid)
   }
 
+  get lastCatchedEventError() {
```

Comment:
IMHO `watchEventsError` would be learer.

Reply:
a `getter` called ==> `watchEventsError` ?

Reply:
I see what you mean, but this is `watchEvents`' error…

Reply:
Okey :) so `watchEventsError` that return `lastCatchedEventError`

Reply:
Let's keep the private field's name in sync with its getter.

Reply:
I meant keep them synchronized with the new name :stuck_out_tongue: